cwms-data-api/src/main/java/cwms/cda/data/dto/location/kind/Outlet.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2024. Hydrologic Engineering Center (HEC).
+ * United States Army Corps of Engineers
+ * All Rights Reserved. HEC PROPRIETARY/CONFIDENTIAL.
+ * Source may not be released without written approval from HEC
+ */
+
+package cwms.cda.data.dto.location.kind;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.data.dto.Location;
+import cwms.cda.data.dto.LocationIdentifier;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.annotations.FormattableWith;
+import cwms.cda.formatters.json.JsonV1;
+
+@FormattableWith(contentType = Formats.JSONV1, formatter = JsonV1.class, aliases = {Formats.DEFAULT, Formats.JSON})
+@JsonDeserialize(builder = Outlet.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+public class Outlet extends ProjectStructure
```

Comment:
I'll add a link to this on my DAO tests.  From the DTO's standpoint it can definitely support base, sub, and aliased locations, since the DTO is dumb.

Reply:
@RyanM-RMA  - Did you add DTO unit test coverage for base location outlets?

Reply:
I did not, it doesn't do any logic for base vs. sub locations, it's just a string in the DTO.  I can add that if you really think it's useful, but I don't think it provides any benefit.  I think that's more of an integration test, since the db does that logic.

Reply:
move this conversation to the new PR